Makefile

```diff
@@ -96,25 +96,28 @@ help: ## Show this help screen
 .PHONY: all
 all: manager ## Build all binaries
 
-$(GO-LICENSES):
+$(TIMESTAMPS_DIR)/go-licenses-installed:
 	go install github.com/google/go-licenses@latest
+	@mkdir -p $(TIMESTAMPS_DIR) && touch $@
+
+go-licenses: $(TIMESTAMPS_DIR)/go-licenses-installed
 
-licenses.csv: $(GO_SOURCES) go.mod ## Track licenses in a CSV file
+licenses.csv: $(TIMESTAMPS_DIR)/go-licenses-installed go.mod ## Track licenses in a CSV file
 	@echo "Tracking licenses into file $@"
 	@echo "========================================"
-	$(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > $@
+	GOOS=linux GOARCH=amd64 $(GO_LICENSES) csv --include_tests $(BASE_GO_PACKAGE)/... > $@
 
 .PHONY: check-licenses
 check-licenses: licenses.csv  ## Check licenses
 	@echo "Checking licenses not to be: $(DISALLOWED_LICENSES)"
 	@echo "============================================"
-	$(GO_LICENSES) check --include_tests $(BASE_GO_PACKAGE)/... \
+	GOOS=linux GOARCH=amd64 $(GO_LICENSES) check --include_tests $(BASE_GO_PACKAGE)/... \
 	--disallowed_types $(DISALLOWED_LICENSES)
 	@echo "--------------------"
 	@echo "Licenses check: PASS"
 
 .PHONY: unit-test
-unit-test: check-licenses
+unit-test: licenses.csv
```

Comment:
Would that pre-requirement be always fulfilled? The `licenses.csv` seems to be committed to the repo.

Reply:
I think it's better to run this check on the CI and if the licence file is different from the one in the PR we mark this check as "failed" and block the PR from being merged

Reply:
Thanks @igor-karpukhin , that makes sense I will fix it and make the check in the CI

@slaskawi I will use the CI, as Igor suggests, to ensure we do not forget to run make licenses.csv and not run on unit tests.

Reply:
Unfortunately, we cannot rely on the CI for reminding us to update the licenses .CSV because it reports differences with the same command just run from your Mac.

Not sure how to proceed now.

Reply:
The check is now on the file dates, that should be enough as a reminder.
I am going to add a comment to explain why we are not comparing generated files instead.

Reply:
I'm OK with this approach.